sensors/nxt_i2c_sensor_defs.c

```diff
@@ -248,6 +248,19 @@ static void mi_xg1300l_remove_cb(struct nxt_i2c_sensor_data *data)
 	}
 }
 
+/*
+ *  convert raw data of HiTechnic Angle Sensor to degrees
+ */
+static int ht_angle_scale(void *context, struct lego_sensor_mode_info *mode_info,
+			u8 index, long int *value)
+{
+	
+	*value = (2* mode_info->raw_data[index] + mode_info->raw_data[index+1]);
```

Comment:
need a space before the `*` here. Could also be written `(mode_info->raw_data[index] << 1)` instead.